chef_master/source/deprecations_locale_lc_all.rst

```diff
@@ -0,0 +1,47 @@
+=======================================================================
+Deprecation: Deprecation of lc_all from locale resource (CHEF-27)
+=======================================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/deprecations_locale_lc_all.rst>`__
+
+.. tag deprecations_locale_lc_all
+
+Setting *LC_ALL* variable is NOT recommended and can break Chef's parsing of command output in unexpected ways. Use one of the more specific *LC_* properties as needed.
+
+.. end_tag
+
+This deprecation warning was added after Chef 15.0.185. Support for property `lc_all` will likely be removed by Chef 15.1.0.
+
+
+Remediation
+=======================================================================
+
+Though it is inadvisable, still if there is a need to set *LC_ALL*, you may set this environment variable manually into any one of the files listed below.
+
+.. code-block:: ruby
+
+	export LC_ALL="<locale_name>"
+
+1. /etc/default/locale
+2. /etc/sysconfig/i18n
+3. /etc/environment
+
+.. note:: List of files is arranged priority wise. The first file exists on the system should be edited.
+
+You can also use *file* resource and add this variable in any other file of your choice and then can source that file to reflect changes.
+
+.. code-block:: ruby
+
+	file "<path_to_file>" do
+	  content "LC_ALL=<locale_name>"
+	end
+
+
+To check the locale value, run
+
+.. code-block:: ruby
+
+	locale -v
+
+.. warning::
+	- Changes on any other file might only be applicable till current session of shell.
```

Comment:
drop this warning. this is probably a step too far in trying to teach unix than provide a workaround for the deprecation of the `lc_all` property.

Reply:
True. Updated